python/generate_document.py

```diff
@@ -0,0 +1,52 @@
+"""
```

Comment:
I don't think you intended this to be part of this pull request? It's a good sample for how to do something with the API though :-)

Reply:
Again, bad clean-up from my end.
Hope you get to use the other suggestions I sent.